js/projection/pg-chain.js

```diff
@@ -0,0 +1,24 @@
+import _ from 'underscore';
+
+class ProjectionChain extends Backbone.Model {
+			initialize(options) {
+				this.projections = [];
+				this.on('change', () => this.update());
+			}
+
+			update() {
+				return _.reduce(this.projections, 
+					(memo, projection) => projection(memo, this.attributes),
+					{}
+					);
```

Comment:
You can make them in 1 line with a shorter name
return _.reduce(this.projections,  (memo, proj) => proj(memo, this.attributes), {});